redbot/cogs/general/general.py

```diff
@@ -267,7 +267,13 @@ async def serverinfo(self, ctx, details: bool = False):
             data.add_field(name=_("Voice Channels"), value=voice_channels)
             data.add_field(name=_("Roles"), value=humanize_number(len(guild.roles)))
             data.add_field(name=_("Owner"), value=str(guild.owner))
-            data.set_footer(text=_("Server ID: ") + str(guild.id))
+            data.set_footer(
+                text=_("Server ID: ")
+                + str(guild.id)
+                + _(' | Use "{command}" for more info on the server.').format(
```

Comment:
Another suggestion after discussion/voting in Discord.
```suggestion
                + _('  •  Use {command} for more info on the server.').format(
```